conf/template/yarp_plugin_device.cpp.in

```diff
@@ -17,9 +17,6 @@
 #  define YARP_PLUGIN_EXPORT YARP_HELPER_DLL_EXPORT
 #endif
 
-using namespace yarp::os;
-using namespace yarp::dev;
```

Comment:
@drdanz This removal broke all the device for which the TYPE argument of the `yarp_add_plugin` macro is specified without the explicit `yarp::dev` namespace, such as https://github.com/robotology/yarp/tree/master/example/externalmodules . It is definitly the right thing to do, but perhaps we could mention this in the ChangeLog?